src/components/icon/assets/crossInCircleEmpty.svg

```diff
@@ -0,0 +1,9 @@
+<!--
+  - Copyright OpenSearch Contributors
+  - SPDX-License-Identifier: Apache-2.0
+  -->
+
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
```

Comment:
Based on @AMoo-Miki 's suggestion: https://github.com/opensearch-project/oui/pull/1014#discussion_r1313465116, let's remove the `width` and `height` props.